python/tests/unit_tests/test_client.py

```diff
@@ -2069,3 +2069,74 @@ def test_evaluate_methods() -> None:
     eval_args = set(inspect.signature(aevaluate).parameters).difference({"client"})
     extra_args = client_args - eval_args
     assert not extra_args
+
+
+@patch("langsmith.client.requests.Session")
+def test_create_run_with_zstd_compression(mock_session_cls: mock.Mock) -> None:
```

Comment:
would also add a test to ensure that the case where the info endpoint does not support `zstd_compression_enabled` uses regular multipart

Reply:
Additoinally, need a test case for `DISABLE_RUN_COMPRESSION`